docs/articles/introduction-to-IoT-security.md

```diff
@@ -0,0 +1,117 @@
+
+Privacy is also a major concern. Many IoT systems provide very deep and near-real time insight into industrial and business processes, as well as into homes and the immediate personal environment. Leakage of personal data can be embarrassing (and costly), but might also be life-threatening; if the whereabouts of a vulnerable or important person is easily tracked then that person becomes a sitting target. 
+
+The purpose of this document is not to describe how to protect information, devices, and other components in an IoT system, but rather to illustrate points that you must consider when implementing IoT security.
+
+## Understanding possible attack surfaces
+
+A good starting point for considering IoT security is to ensure that you are aware of the potential points of weakness in the system. The following diagram illustrates a generic IoT architecture, encompassing many devices interacting with a set of monitoring and control services running in the cloud.
+
+In this diagram, devices are the individual items of hardware that are located in the field. They can have varying levels of complexity, but typically report status information and may respond to commands. A field gateway is a device, appliance, or more general-purpose server that acts as a communications enabler, device control system, or local data processing hub. As an example, a smart car could be considered to be a field gateway with its own local control software gathering data from individual devices located on the vehicle, monitoring them, and sending commands back to them. A field gateway, or individual devices, might need to communicate with a remote control system that controls and coordinates actions across different sites. This could be achieved by connecting to the services that implement this control system directly or through a cloud gateway. A cloud gateway can act as a regional hub implementing some autonomous regional services. The connection between a field gateway and/or local devices with a cloud gateway could be static or dynamic. For example, a field gateway that roams (such as a smart car) might switch to different cloud gateways as it changes location.
```

Comment:
This seems like a lot of introductory material isn't directly related to security. I would prefer to tigten this up and link to supporting content.